DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Utilities/Utility.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
 
+using DNNConnect.CKEditorProvider.Constants;
```

Comment:
(nit) Can this get moved inside the `namespace` declaration, please?

Reply:
Sure, I'll fix that. 

I noticed changes like this recently. Do you know the benefits of that?

Reply:
It's primarily a style rule, but it does have some minor benefits ([described in the StyleCop rule docs](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/fe00f94f0ba5471eb9e3e752a3ae37bf19e5f2e3/documentation/SA1200.md))

Reply:
That makes sense, thanks.